controllers/defaults/resources.go

```diff
@@ -93,6 +93,26 @@ var (
 				corev1.ResourceMemory: resource.MustParse("1Gi"),
 			},
 		},
+		"crashcollector": {
```

Comment:
See [this example](https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph-cluster/values.yaml#L326-L331) for crash collector resources.